Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -102,14 +102,11 @@ def find_instance_method_call(node, instance, method_name)
 
       # Returns nil if does not depend on dependency_name
       # args: node - dependency_name - dependency's name
-      def depends_on?(dependency_name)
+      def depends_on?(dependency_name, *types)
```

Comment:
Okay will make sure for it to not happen again. 
Found it a bit tiresome to commit only few lines of a file. 😅